reference/conanfile/methods/configure.rst

```diff
@@ -0,0 +1,60 @@
+.. _reference_conanfile_methods_configure:
+
+configure()
```

Comment:
We can add a link to the tutorial: https://docs.conan.io/en/2.0/tutorial/creating_packages/configure_options_settings.html